src/App.css

```diff
@@ -13,15 +11,16 @@
   }
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.App {
+  width: 50vw;
```

Comment:
- [ ] Based on the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) your web application should be mobile-friendly, please make your design look like the Behance design on mobile you can use the developer tools to see how your website looks on mobile screen sizes when designing 👍🏿  
See video below ⬇️ 

https://user-images.githubusercontent.com/69638013/175778378-8a4442fd-e5b3-4cf2-8593-7e7cc0741eaa.mp4